### ISSUE TITLE
Rotary encoder help

### ISSUE THREAD

**Issue by imarkone** [Maschine2501/NR1-UI#13]
Hello, I have a problem with the rotary encoder, the switch does not work on it. I tried two models, changed cables and nothing. The rotation works OK, but when I press the rotary encoder for a short time or longer, nothing happens. 
Thanks for help. M
Volumio version is  3.233

**Comment by imarkone** [comment]
I tried using bash command NR1-UI/PreConfiguration.sh and tried custom setting for rotary encoder, also tried other GPIO for SW button but nothing helped.. I use HW-040 and Keyes. Both behave the same, gpio 27 switch does not work.

**Comment by imarkone** [comment]
I managed to solve the problem, I noticed that both encoder modules are missing a 10K smd resistor for the switch. I also looked at other encoders in e-shops, and even there, according to the pictures, the resistor is missing, but I also found some that had a resistor, e.g. on ebay. So I added it, I didn't have an smd, but a resistor is a resistor :-) After adding a resistor, the switch already works, I guess it will help someone
![IMG_2077](https://user-images.githubusercontent.com/50739478/185493041-3c8c98b8-c52e-4035-bc16-a3a3c0cee827.jpg)
![IMG_2076](https://user-images.githubusercontent.com/50739478/185493061-8b0c4f82-002f-491e-b41b-4efe5e7b141d.jpg)
.
